docset/winserver2022-ps/clusterawareupdating/Add-CauClusterRole.md

```diff
@@ -17,35 +17,39 @@ Adds the CAU clustered role that provides the self-updating functionality to the
- [-CauPluginArguments <Hashtable[]>] [-MaxFailedNodes <Int32>] [-MaxRetriesPerNode <Int32>]
- [-NodeOrder <String[]>] [-PreUpdateScript <String>] [-PostUpdateScript <String>] [-ConfigurationName <String>]
- [-RequireAllNodesOnline] [-WarnAfter <TimeSpan>] [-StopAfter <TimeSpan>] [-RebootTimeoutMinutes <Int32>]
- [-SeparateReboots] [-RunPluginsSerially] [-StopOnPluginFailure] [-EnableFirewallRules]
- [-FailbackMode <FailbackType>] [-SuspendClusterNodeTimeoutMinutes <Int32>] [[-ClusterName] <String>]
- [[-Credential] <PSCredential>] [-Force] [-WhatIf] [-Confirm] [<CommonParameters>]
+Add-CauClusterRole [-VirtualComputerObjectName <String>] [-GroupName <String>]
+[-StartDate <DateTime>] [-DaysOfWeek <Weekdays>] [-WeeksOfMonth <Int32[]>]
+[-CauPluginName <String[]>] [-CauPluginArguments <Hashtable[]>] [-MaxFailedNodes <Int32>]
+[-MaxRetriesPerNode <Int32>] [-NodeOrder <String[]>] [-PreUpdateScript <String>]
+[-PostUpdateScript <String>] [-ConfigurationName <String>] [-RequireAllNodesOnline]
+[-WarnAfter <TimeSpan>] [-StopAfter <TimeSpan>] [-RebootTimeoutMinutes <Int32>] [-SeparateReboots]
+[-RunPluginsSerially] [-StopOnPluginFailure] [-EnableFirewallRules] [-FailbackMode <FailbackType>]
+[-SuspendClusterNodeTimeoutMinutes <Int32>] [[-ClusterName] <String>] [[-Credential] <PSCredential>]
+[-Force] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ### Weekly
 ```
-Add-CauClusterRole [-VirtualComputerObjectName <String>] [-GroupName <String>] [-StartDate <DateTime>]
- [-DaysOfWeek <Weekdays>] [-IntervalWeeks <Int32>] [-CauPluginName <String[]>]
- [-CauPluginArguments <Hashtable[]>] [-MaxFailedNodes <Int32>] [-MaxRetriesPerNode <Int32>]
- [-NodeOrder <String[]>] [-PreUpdateScript <String>] [-PostUpdateScript <String>] [-ConfigurationName <String>]
- [-RequireAllNodesOnline] [-WarnAfter <TimeSpan>] [-StopAfter <TimeSpan>] [-RebootTimeoutMinutes <Int32>]
- [-SeparateReboots] [-RunPluginsSerially] [-StopOnPluginFailure] [-EnableFirewallRules]
- [-FailbackMode <FailbackType>] [-SuspendClusterNodeTimeoutMinutes <Int32>] [[-ClusterName] <String>]
- [[-Credential] <PSCredential>] [-Force] [-WhatIf] [-Confirm] [<CommonParameters>]
+Add-CauClusterRole [-VirtualComputerObjectName <String>] [-GroupName <String>]
+[-StartDate <DateTime>] [-DaysOfWeek <Weekdays>] [-IntervalWeeks <Int32>]
+[-CauPluginName <String[]>] [-CauPluginArguments <Hashtable[]>] [-MaxFailedNodes <Int32>]
+[-MaxRetriesPerNode <Int32>] [-NodeOrder <String[]>] [-PreUpdateScript <String>]
+[-PostUpdateScript <String>] [-ConfigurationName <String>] [-RequireAllNodesOnline]
+[-WarnAfter <TimeSpan>] [-StopAfter <TimeSpan>] [-RebootTimeoutMinutes <Int32>] [-SeparateReboots]
+[-RunPluginsSerially] [-StopOnPluginFailure] [-EnableFirewallRules] [-FailbackMode <FailbackType>]
+[-SuspendClusterNodeTimeoutMinutes <Int32>] [[-ClusterName] <String>] [[-Credential] <PSCredential>]
+[-Force] [-WhatIf] [-Confirm] [<CommonParameters>]
```

Comment:
You need a single space for each line after the first line in the syntax block. You can do this in the next PR since this is just for reflow.

Reply:
Will do, thanks @sdwheeler